### ISSUE TITLE
code

### ISSUE THREAD

**Issue by morefruits** [awccnet/AWCC-Net#3]
Hello author, thank you for your excellent work. May I ask if you could share the loss function and training code? Thank you！
